Class-10-(Scope-Exceptions-Types).md

```diff
@@ -0,0 +1,23 @@
+## In the prep work this week, the students learned:
+1. More about scope. Scope was covered briefly in the Functions chapter, however, now students have covered it more in-depth in terms of the order code executes in.
+1. What exceptions are.
+1. How to raise an exception.
+1. More about what a primitive data type is.
+
+## Announcements
+1. More info to come... when in doubt, check with your class Candidate Engagement Manager!
+
```

Comment:
The third graded assignment opens after this class. Feel free to start working on the smaller parts.

Reply:
Done! Thanks!